app/public/js/profile.js

```diff
@@ -106,12 +106,16 @@ function getCountryElement()
 
 function updateRegionOptions()
 {
-	var country_id = parseInt(getCountryElement().val());
-	var $datalist = $('#regions');
-	$datalist.empty();
+	let country_id = parseInt(getCountryElement().val());
```

Comment:
Keep the 'var'.  It may seem paranoid to worry about browsers that don't support 'let' when it has been popular for several years but I see no reason to make IE stop working just so we can use later JavaScript language features like 'let'.  I want users of old web browsers to still be ok even if we don't test much with their browsers.